Applications/DataExplorer/DataView/DirectConditionGenerator.cpp

```diff
@@ -30,36 +30,46 @@
 #include <cmath>
 #include <limits>
 
-const std::vector< std::pair<std::size_t,double> >& DirectConditionGenerator::directToSurfaceNodes(const MeshLib::Mesh &mesh, const std::string &filename)
+const std::vector<std::pair<std::size_t, double>>&
+DirectConditionGenerator::directToSurfaceNodes(const MeshLib::Mesh& mesh,
+                                               const std::string& filename)
 {
     if (_direct_values.empty())
     {
-        GeoLib::Raster* raster (GeoLib::IO::AsciiRasterInterface::readRaster(filename));
-        if (! raster) {
-            ERR("Error in DirectConditionGenerator::directToSurfaceNodes() - could not load raster file.");
+        GeoLib::Raster* raster(
+            GeoLib::IO::AsciiRasterInterface::readRaster(filename));
+        if (!raster)
+        {
+            ERR("Error in DirectConditionGenerator::directToSurfaceNodes() - "
+                "could not load raster file.");
             return _direct_values;
         }
 
-        const MathLib::Vector3 dir(0,0,-1);
-        const std::vector<GeoLib::Point*> surface_nodes(MeshLib::MeshSurfaceExtraction::getSurfaceNodes(mesh, dir, 90) );
+        const MathLib::Vector3 dir(0, 0, -1);
+        const std::vector<MeshLib::Node*> surface_nodes(
```

Comment:
This will cause problem later in visualisation because you can't show mesh nodes without elements.